tests/AbstractCommandTest.php

```diff
@@ -276,5 +287,57 @@ public function end(string $token, ContextInterface|array $context = []): void
         $db->setProfiler($profiler);
 
         $db->createCommand($sql)->execute();
+        $db->close();
+    }
+
+    public function testBindParamsOverflowIssue(): void
+    {
+        $db = $this->getConnection();
+
+        if ($db->getDriverName() !== 'pgsql') {
+            $this->markTestSkipped('Test is intended for use with pgsql database.');
```

Comment:
Should it be moved to PostgreSQL package?

Reply:
Absolutely it should be overridden in each DBMS package, returning the corresponding value. But for now for simplicity, as a temporary mock, in order to make the tests run without modifying db-pgsql package I put it here.